src/content/lib/WorkerProfileContent/GeneralTab/GeneralTab.js

```diff
@@ -1,36 +1,46 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import { Field } from 'redux-form'
-import { TextField, SelectField } from 'redux-form-material-ui'
-import { MenuItem } from 'material-ui'
-import { Icon } from 'src/components/common'
+import { TextField } from 'redux-form-material-ui'
```

Comment:
import { TextField } from 'redux-form-material-ui-next'

we use new components